src/components/dialog/index.js

```diff
@@ -88,6 +89,14 @@ class Dialog extends BaseComponent {
     width: '90%'
   };
 
```

Comment:
let's just migrate the Dialog in places we're using it 
like `PickerDialog.android` and `PickerDialog.ios`

Reply:
Done